docs/.project/INTEGRATION_TEST.md

```diff
@@ -120,6 +121,11 @@ Based on the [test diagram](./test.bpmn.png):
 
 - [ ] Open [`broken.bpmn`](./broken.bpmn) and verify a proper error message is shown (_No diagram to display_)
 
+#### Accessibility
+
+- [ ] No new issues were reported in the devtools console
+- [ ] Running `window.__A11Y_CHECKER__.getViolations()` returns an empty array
```

Comment:
I think this could be a button in the UI?

Reply:
I could add it, sure.

Reply:
https://github.com/bpmn-io/camunda-modeler-plugin-a11y-checker/issues/1